Adds basic documentation and example for external access integrations & secrets (Snowpark concept) (#5262)
dbt-labs/docs.getdbt.com: website/docs/docs/build/python-models.md

    )
```

> **External Access Integrations and Secrets**

Reviewer: ```suggestion
**External access integrations and secrets**
```

> If you want to use Snowflake’s [external access]([url](https://docs.snowflake.com/en/developer-guide/external-network-access/external-network-access-overview)) together with [secrets]([url](https://docs.snowflake.com/en/developer-guide/external-network-access/secret-api-reference)) to query external APIs within dbt Python models there are some additional configurations to use:

Reviewer: ```suggestion
To query external APIs within dbt Python models, use Snowflake’s [external access](https://docs.snowflake.com/en/developer-guide/external-network-access/external-network-access-overview) together with [secrets](https://docs.snowflake.com/en/developer-guide/external-network-access/secret-api-reference). Here are some additional configurations you can use:
```
